README.md

```diff
@@ -28,6 +28,7 @@ local osys = require("cmake-tools.osys")
 require("cmake-tools").setup {
   cmake_command = "cmake", -- this is used to specify cmake command path
   ctest_command = "ctest", -- this is used to specify ctest command path
+  ctest_extra_args = {}, -- this is used to specify extra args for ctest
```

Comment:
Maybe we can put this variable into `Config.base_settings`? So we can edit it easily.

Reply:
I like the idea. Done.